feature: include SonarQube's scanner on pr checks

MikelAlejoBR opened the pull request:
Includes SonarQube's scanner to be run along with the other stuff in the pr checks. 

[[RHCLOUD-16840]](https://issues.redhat.com/browse/RHCLOUD-16840)

MikelAlejoBR committed:
feature: include SonarQube's scanner on pr checks

lindgrenj6 commented on sonarqube.bash:
```diff
@@ -0,0 +1,87 @@
+  -import \
+  -keystore "$rh_it_keystore_file" \
+  -noprompt \
+  -storepass "$rh_it_keystore_pass"
```
:sweat:

MikelAlejoBR replied:
Yep...

lindgrenj6 commented on sonarqube.bash:
```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```
What would you think about hiding this file in a `scripts/` folder or something? Like I can understand having `pr_check.sh` and `build_deploy.sh` in the root since jenkins needs them - but this one doesn't need to be in the root necessarily?

MikelAlejoBR replied:
Yeah, I agree. Moving it now!

lindgrenj6 reviewed (commented):
Looks good with one minor thing - nbd in case I'm crazy.

MikelAlejoBR committed:
refactor: move the SonarQube script to the "scripts" directory

MikelAlejoBR committed:
refactor: don't source "sonarqube" script on "pr_checks"

Now two pipelines will be launched when a PR gets some activity: one for
the integration tests and another one for the SonarQube scan. Therefore,
there is no need to source the SonarQube scanner's script into the "pr
check" one.

MikelAlejoBR committed:
refactor: change "sonarqube" script's permissions to 755

This way the CI pipeline can execute the script without having to issue
another command to change the permissions on the file.

MikelAlejoBR committed:
fix: add dummy artifact for the pipeline to succeed

lpichler reviewed (approved):
👏
